server/datastore/mysql/software_test.go

```diff
@@ -6106,9 +6237,8 @@ func testListHostSoftwareVulnerabileAndVPP(t *testing.T, ds *Datastore) {
 	// should only return "file1" and "vpp1"
```

Comment:
```suggestion
	// should only return "file1" (vpp1 is not compatible with the platform)
```

This is changed due to platform filtering, right?